sql/drop.go

```diff
@@ -0,0 +1,48 @@
+package sql
+
+import (
+	"fmt"
+
+	"github.com/cockroachdb/cockroach/keys"
+	"github.com/cockroachdb/cockroach/sql/parser"
+	"github.com/cockroachdb/cockroach/structured"
+)
+
+// DropTable drops a table.
+func (p *planner) DropTable(n *parser.DropTable) (planNode, error) {
```

Comment:
Ah, then drop should probably do something else then. either rename the descriptor, or add a way to mark it as deleted.
Still, whether the data gets deleted or not, we still need a test that checks for correct metadata manipulation.